doc/changelog/08-tools/13460-master+fix-ocaml-int-extraction-clash.rst

```diff
@@ -0,0 +1,9 @@
+- **Fixed:**
+  Name :n:`int` can be used in extraction without conflicting with
+  OCaml's :n:`int` type
```

Comment:
You should add that when wanting to extract to ocaml's builtin int the alias should be used, and the alias should be mentioned in the non-changelog part of the refman.

Reply:
I added a line in the change log, is it ok?

I otherwise see this PR as a workaround which should be extended to any conflicting name, not only `int`. I'm not opposed to document the current workaround in the reference manual, but I'm not sure this is the long term to go.

Note that `Extract Constant`/`Extract Inductive` is itself pretty fragile since it is uninterpreted text. I believe that what I proposed in this [comment](https://github.com/coq/coq/pull/13460#issuecomment-732813747) is a better solution. In particular, we could consider it for 8.14. I did not do it because it would have taken me more time to understand where to exactly put the corresponding table, but if someone is willing to provide help, I can try at some time.